docs/guides/open-telemetry-collector.md

```diff
@@ -65,6 +65,19 @@ service:
 ```
 
 ## Configure OTLP
+You can configure OTLP-related flags in a yaml file.
```

Comment:
Can we just say `You can configure OTLP-related flags in the config file`. `in a yaml file` sounds a bit confusing as I thought it meant another yaml file for OTLP configs